tests/functional/test_matrix_service.py

```diff
@@ -57,10 +60,25 @@ def test_matrix_service(self):
         # analyze produced matrix against standard
         self._analyze_matrix_results(request_id)
 
-    def _post_matrix_service_request(self):
+    def test_matrix_service_ss2(self):
+        timeout = int(os.getenv("MATRIX_TEST_TIMEOUT", 300))
+        num_bundles = int(os.getenv("MATRIX_TEST_NUM_BUNDLES", 400))
+        bundle_fqids = json.loads(open(f"{self.res_dir}/ss2_2536.json", "r").read())[:num_bundles]
+
+        request_id = self._post_matrix_service_request(bundle_fqids, "zarr")
+
+        # wait for request to complete
+        time.sleep(2)
+        WaitFor(self._poll_get_matrix_service_request, request_id)\
+            .to_return_value(MatrixRequestStatus.COMPLETE.value, timeout_seconds=timeout)
+
+        # TODO: analyze results
+        # self._analyze_matrix_results(request_id)
```

Comment:
Currently this test is just an easy way to configure the size of the pancreas data to test. Any ideas on how we can do some results validation?

Reply:
Easiest thing would be to check that the number of rows in the output matrix matches `num_bundles`. That would actually catch all the errors we've observed thus far.